index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <h1 id = "title"> Survey Form </h1>
+  </head>
+  <body>
+<p id = "description"> Description </p>
```

Comment:
Kindly make use of the HTML semantic tags, you can read more about it in your dashboard "HTML & CSS best practices" and apply them.